assets/js/modules/reader-revenue-manager/components/setup/SetupMain.stories.js

```diff
@@ -19,29 +19,99 @@
 /**
  * Internal dependencies
  */
+import {
+	provideModuleRegistrations,
+	provideModules,
+} from '../../../../../../tests/js/utils';
+import { CORE_FORMS } from '../../../../googlesitekit/datastore/forms/constants';
+import {
+	MODULE_SLUG,
+	MODULES_READER_REVENUE_MANAGER,
+	READER_REVENUE_MANAGER_SETUP_FORM,
+	SHOW_PUBLICATION_CREATE,
+} from '../../datastore/constants';
+import ModuleSetup from '../../../../components/setup/ModuleSetup';
 import WithRegistrySetup from '../../../../../../tests/js/WithRegistrySetup';
-import { MODULES_READER_REVENUE_MANAGER } from '../../datastore/constants';
-import SetupMain from './SetupMain';
+import { publications } from '../../datastore/__fixtures__';
 
 function Template() {
-	return <SetupMain />;
+	return <ModuleSetup moduleSlug={ MODULE_SLUG } />;
 }
 
-export const Default = Template.bind( {} );
-Default.storyName = 'Default';
-Default.scenario = {
+export const NoPublications = Template.bind( {} );
+NoPublications.storyName = 'No publications';
+NoPublications.args = {
+	setupRegistry: ( registry ) => {
+		registry
+			.dispatch( MODULES_READER_REVENUE_MANAGER )
+			.receiveGetPublications( [] );
+	},
+};
+NoPublications.scenario = {
+	// No parameters are needed for this scenario, but we still need to define the scenario object to include the story in the VRT suite.
```

Comment:
I don't feel like this comment is needed. As time goes on, we should have the collective knowledge that the parameter doesn't need that label or anything. While this is useful comment now, it's also kind of repetitive and bloat-y.